cmd/flowrunner/testdata/flows/two_questions_test.json

```diff
@@ -49,7 +49,7 @@
               }
             ],
             "created_on": "2000-01-01T00:00:00.000000000-00:00",
-            "text": "Hi Ben Haggerty! What is your favorite color? (red/blue) Your number is +12065551212",
+            "text": "Hi Ben Haggerty! What is your favorite color? (red/blue) Your number is tel:+12065551212",
```

Comment:
Need to think about migrating URN references to something like `@(format_urn(...))`

Reply:
Ya.